### DISCUSSION TITLE
Welcome to COMP/CON Discussions!

### DISCUSSION THREAD

**Discussion by jarena3** [massif-press/compcon#2044] [Archive]
Hello Everyone,
Over the next few weeks and months I will be trying to move the bulk of COMP/CON related discussion, from Discord/Reddit/Twitter/etc to the C/C GitHub repository's Discussions board, and better use the community and project management side of GitHub to centralize all the various posts, updates, questions, comments, etc. in one single, easily accessible, place.

Please bear with me as I get these plates spinning, and can pare down my presence elsewhere to focus on this space.

Additionally, please be mindful of the project's [Code of Conduct](https://github.com/massif-press/compcon/issues) and [GitHub's Community Guidelines](https://docs.github.com/articles/github-community-guidelines). Anything posted here is subject to moderation by project contributors at their discretion. Violators of these guidelines, as well as those who post in poor taste etc. will be removed.

**Comment by jarena3** [comment]
## Useful Links:

[COMP/CON Milestones](https://github.com/massif-press/compcon/milestones) - The COMP/CON roadmap, showing what issues are being worked on, their status, and what's coming in future development
[The COMP/CON Wiki](https://github.com/massif-press/compcon/wiki) - A repository for useful COMP/CON related information. Contributors welcome!
[The Massif Press Projects Board](https://github.com/orgs/massif-press/projects?type=beta) - An index of in-progress and upcoming LANCER-related app and resource development

**Comment by wsethbrown** [comment]
Thanks for putting this together. Will be having a look through the repo while on vacation and will put any thoughts in the discussion!
